### PR TITLE
Updated the README.md

### PR DESCRIPTION
**Pull request by yaseminbridges**
Updated the README.md to better introduce the project with some example usage

### CODE REVIEW

File: README.md
Comment (julesjacobsen) [inline README.md:79]: This link leads to a 404 :(

**Review by julesjacobsen — CHANGES_REQUESTED** [review]
Looks great. Can you update or fix the broken link on the last line of the README before merging.

File: README.md
Comment (yaseminbridges) [reply README.md:79]: This link will work once #348 is merged

**Comment by julesjacobsen** [comment]
OK - please merge this once the tox conflict is fixed in #348

**Pull request merged**
